README.md

```diff
@@ -136,21 +140,68 @@ See the [CONTRIBUTING](https://github.com/OpenTermsArchive/contrib-declarations/
 
 ### Installing
 
-This module is built with [Node](https://nodejs.org/en/). You will need to [install Node >= v14.x](https://nodejs.org/en/download/) to run it.
-
-Clone the repository and install dependencies:
+- This module is built with [Node](https://nodejs.org/en/). You will need to [install Node >= v14.x](https://nodejs.org/en/download/) to run it.
+- When refering to the root folder, it means the folder where you will be `git pull`ing everything.
+
+#### Declarations repository
+
+1. Locally clone your declaration repository, e.g., `git@github.com:OpenTermsArchive/contrib-declarations.git`.
+2. Go into your folder and initialize it, e.g., `cd contrib-declarations; npm install`.
+3. You can now modify your declarations in the `./declarations/` folder, following [these instructions](https://github.com/OpenTermsArchive/contrib-declarations/blob/main/CONTRIBUTING.md).
+4. When you want to test:
+    - If you want to test every declaration, run `npm test`.
+    - If you want to test a specific declaration, run `npm test <name of the service>`, e.g., `npm test HER`.
+    - If you want to only test the integrity of a specific declaration, run `npm test -- --schema-only <name of the service>`, e.g., `npm test -- --schema-only HER`.
+5. Once you have done that, if you have any error, it will be prompted and detailed at the end of the test.
+    - E.g., `InaccessibleContentError`: Your selector is wrong and should be fixed.
+    - E.g., `TypeError`: The file declaration is invalid.
+    - E.g., if you have a weird error, you may want to contact OTA, if may be a bug.
```

Comment:
I am not sure of the added value of this part 🙂

Reply:
I mainly put that in our side to get more insights about errors - let me know if I should rather remove that part.

As previously discussed, this may be too many details for an internal usage indeed.

Reply:
If this was not clear enough and granted writing documentation, I believe this should lead to improving the test failure messages instead 🙂

I would prefer that we remove this from the doc. If you feel like trying to improve the logging messages, feel free to open a PR improving that file: https://github.com/ambanum/OpenTermsArchive/blob/master/scripts/validation/validate.js 👍